web/styles/subscriptions.css

```diff
@@ -968,6 +968,14 @@ div.settings-radio-input-parent {
     }
 }
 
+.multiuse_radio_section .settings-radio-input-parent {
```

Comment:
This should be moved to the file which has other CSS for invite modal, which I guess is `zulip.css`.